src/components/accounts/RecoverWithLink.js

```diff
@@ -201,7 +206,10 @@ class RecoverWithLink extends Component {
                             <Title>{translate('recoverWithLink.errorTitle')}</Title>
                             <Desc>{translate('recoverWithLink.errorP')}</Desc>
                             {!DISABLE_CREATE_ACCOUNT &&
-                                <Button onClick={() => history.push('/create')}>
+                                <Button onClick={() => {
+                                    Mixpanel.track("IE Click create button")
```

Comment:
This button only shows if the recovery link is invalid/expired: `IE with link expired click create button`